src/styles/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  position: absolute;
+  margin: 0 auto;
+  top: 10px;
+  left: 50px;
```

Comment:
The logo is positioned absolutely, which might cause issues with responsiveness and alignment within the header. Consider using flexbox properties to align the logo within the header container.